kaggle_environments/envs/lux_ai_2022/luxai2022/env.py

```diff
@@ -167,6 +189,9 @@ def _handle_bid(self, actions):
             if k not in self.agents:
                 raise ValueError(f"Invalid player {k}")
             if "faction" in a and "bid" in a:
+                if a["faction"] not in [e.name for e in FactionTypes]:
+                    failed_agents[k] = False
```

Comment:
This is already false, should this be `True`?

Reply:
Silent error here. Should be true, but was silent since this stops the code from initializing some data (which is checked later down the env step function). Fixing